### ISSUE TITLE
Laggy Animation

### ISSUE THREAD

**Issue by rodrigonso** [jamesblasco/modal_bottom_sheet#16]
Hi James,

Thank you for sharing this package, it is something I've been waiting for a long time to implement into my application.

However, I noticed the animation for the generic modal bottom sheet has a very low framerate (laggy). I am not sure if that is an issue caused from my end or not.

This is what the animation looks like on my actual device (not the iOS simulator)

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/46465969/81478736-c8767980-91e4-11ea-8191-99990b74f857.gif)

I'm invoking the modal bottom sheet with the following code:

`showCupertinoModalBottomSheet(
  context: context,
  builder: (context, scrollController) => Container(),
})`

What are your thoughts?

Cheers!

**Comment by jamesblasco** [comment]
Mmmmh most of the logic is based on the original flutter modal bottom sheet, so you should expect a very similar performance. First I would try to test the modal in release mode, also try if the textfield autofocus could be the one making lag.

**Comment by rodrigonso** [comment]
Right, that makes sense. I have created a new test case and wanted to share what I found -- it looks like it won't work well with many text fields.

The gifs below show the tests I made with a different number of text fields. 

Also, the code I used inside the invoking function is as follows:

`showCupertinoModalBottomSheet(
            context: context,
            builder: (context, scrollController) => Scaffold(
              body: Column(
                children: [
                  TextField(
                    onTap: () => null,
                    decoration: InputDecoration(hintText: "Username"),
                  ),
                ],
              ),
            ),
          )`

![ezgif com-video-to-gif (4)](https://user-images.githubusercontent.com/46465969/81489315-92171980-9239-11ea-8fb5-c12f942938f7.gif)
![ezgif com-video-to-gif (3)](https://user-images.githubusercontent.com/46465969/81489316-92171980-9239-11ea-9dc6-560296f20e2d.gif)
![ezgif com-video-to-gif (2)](https://user-images.githubusercontent.com/46465969/81489317-92afb000-9239-11ea-8ff0-3be212f6cec8.gif)
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/46465969/81489318-92afb000-9239-11ea-939b-eb32162a44f0.gif)

Disabling the autofocus also decreases the lag.

Cheers!
